health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/MapViewPopup.js

```
@@ -0,0 +1,325 @@
+      }}
+      footerChildren={[
+        <Button
+          type={"button"}
+          size={"large"}
+          variation={"primary"}
+          label={t("Close")}
+          onClick={() => {
+            setShowPopup(false);
+          }}
+          style={{ width: "12.5rem" }}
+        />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)